packages/http-client-python/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 0.6.6
 
+### Bug Fixes
+
+- Fix sphinx typing for raising documentation
```

Comment:
Since 0.6.6 was already released before this PR, this part of changelog shall be added in new `0.6.7`. @iscai-msft